components/perl/Carp-Clan/pkg5

```diff
@@ -0,0 +1,14 @@
+{
+    "dependencies": [
+        "SUNWcs",
+        "runtime/perl-522",
+        "runtime/perl-524",
+        "runtime/perl-534",
+        "shell/ksh93",
+        "system/library"
+    ],
+    "fmris": [
+        "library/perl-5/carp-clan",
+    ],
```

Comment:
This is the result of the missing -$(PLV) on the fmri.

Reply:
also fixed by inheritance.